lib/oli_web/live/delivery/student/prologue_live.ex

```diff
@@ -11,9 +11,10 @@ defmodule OliWeb.Delivery.Student.PrologueLive do
   alias Oli.Delivery.Sections
   alias Oli.Delivery.Settings
   alias Oli.Publishing.DeliveryResolver, as: Resolver
-  alias OliWeb.Common.FormatDateTime
+  alias Oli.Repo
```

Comment:
We should be careful to not import Oli.Repo into the oli_web domains and liveviews if it's avoidable. Instead a function in the oli domain (`Oli.Delivery.Attempts` perhaps?) would be cleaner